tests/test.py

```diff
@@ -1130,7 +1130,7 @@ def setUp(self):
 
         self.output_filename = 'acra_rollback_output.txt'
         rollback_output_table.create(self.engine_raw, checkfirst=True)
-        if self.TLS_ON:
```

Comment:
or maybe instead of `if` we should use `self.sslmode = SSLMODE` (https://github.com/cossacklabs/acra/pull/129/files#diff-b131fe2646b3e4980d06f90692dc2401L68)

Reply:
decided that it's ok for now